l1-contracts/contracts/state-transition/l2-deps/IL2GenesisUpgrade.sol

```diff
@@ -15,10 +15,16 @@ struct ForceDeployment {
     bytes input;
 }
 
+// solhint-disable-next-line gas-struct-packing
 struct ZKChainSpecificForceDeploymentsData {
     bytes32 baseTokenAssetId;
     address l2LegacySharedBridge;
-    address l2Weth;
+    address predeployedL2WethAddress;
+    address baseTokenL1Address;
+    /// @dev Some info about the base token, it is
```

Comment:
If you already include this, can it be used to set the baseTokenName in L2BaseToken?

Reply:
Discussed internally, considering timeline restrictions, it may be better to not add new features now (even if small, it will require of quite some tests. Also those methods in `L2BaseToken` are marked as unstable and so don't necessarily need to return correct values)